lib/src/flutter_google_places.dart

```diff
@@ -48,7 +48,7 @@ class PlacesAutocompleteWidget extends StatefulWidget {
 
   PlacesAutocompleteWidget(
       {Key? key,
-      required this.apiKey,
```

Comment:
I think should keep `required` keyword here, user can use it like `show(apiKey: null)`